src/js/Pane/Grid.js

```diff
@@ -184,7 +186,76 @@ Wui.Grid.prototype = $.extend(new Wui.Pane(), {
                     // hide the header
                     if(me.hideHeader)    me.heading.height(0);
                 },
-    
+    exportToCsv: function(filename, filterDataItem, filterValue) {
```

Comment:
Please add method header comments for this new code (see [Style Guide rul 5.0 ](http://www.wui-js.com/js-style-guide.html#5_0)